homeassistant/components/device_tracker/tplink.py

```diff
@@ -46,6 +60,45 @@ def get_scanner(hass, config):
 
 
 class TplinkDeviceScanner(DeviceScanner):
+    """Queries the router for connected devices."""
+
+    def __init__(self, config):
+        """Initialize the scanner."""
+        from tplink.tplink import TpLinkClient
+        self.host = config[CONF_HOST]
```

Comment:
Since host isn't used outside this method we can cache it in a local variable, instead of an instance attribute.